cluster-autoscaler/core/scaledown/legacy/legacy_test.go

```diff
@@ -1220,7 +1220,8 @@ func TestScaleDown(t *testing.T) {
 
 func waitForDeleteToFinish(t *testing.T, sd *ScaleDown) {
 	for start := time.Now(); time.Since(start) < 20*time.Second; time.Sleep(100 * time.Millisecond) {
-		if !sd.IsNonEmptyNodeDeleteInProgress() {
+		empty, drained := sd.nodeDeletionTracker.DeletionsInProgress()
+		if len(empty)+len(drained) == 0 {
```

Comment:
The semantics are changing here - previously this function would only wait for a deletion with drain to finish, now it waits for both drain and empty nodes deletions. Does this affect any tests?

Reply:
It doesn't. Reverted to keep previous semantics.